File: games_java/GuessTheNumber/GuessTheNumber.java
package games_java.GuessTheNumber;

import java.util.Scanner;

public class GuessTheNumber {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int num = (int) (Math.random() * 100 + 1);
		int guess = 0;
		while (guess != num) {
			System.out.println("Guess a number 1-100");
			guess = sc.nextInt();

			if (guess == num) {
				System.out.println("You got it!");
			} else if (guess > num) {
				System.out.println("Too high!");
			} else {
				System.out.println("Too low!");
			}
		}
		sc.close();
		System.exit(0);
	}
}

File: games_js/GuessTheNumber/guessTheNumber.js
// the start function gets run when the game starts
async function start() {
	// your code goes here!

	let num = Math.round(random(1, 100));
	let guess;

	while (guess != num) {
		guess = await prompt('What is your guess?');

		if (guess == num) {
			await alert('Correct!');
		} else if (guess > num) {
			await alert('Too high.');
		} else {
			await alert('Too low.');
		}
	}

	exit();
} // end of the start function
